optuna_dashboard/ts/components/GraphParallelCoordinate.tsx

```diff
@@ -169,6 +169,21 @@ const plotCoordinate = (
       .join("")
   }
 
+  const calculateLogScale = (values: number[]) => {
+    const logValues = values.map((v) => {
+      return Math.log10(v)
+    })
+    const minValue = Math.min(...logValues)
+    const maxValue = Math.max(...logValues)
+    const range = [minValue, maxValue]
```

Comment:
Both ends of ticks should be within the range.
```suggestion
    const range = [Math.floor(minValue), Math.ceil(maxValue)]
```